understanding/22/accessible-authentication-minimum.html

```diff
@@ -212,11 +212,10 @@ <h2>Resources</h2>
             <li><a href="https://w3c.github.io/coga/issue-papers/#web-security-and-privacy-technologies">Cognitive Accessibility Issue Papers 4. Web Security and Privacy Technologies</a> and <a href="https://w3c.github.io/coga/issue-papers/privacy-security.html">Web Security and Privacy Technologies</a></li>
             <li><a href="https://www.w3.org/TR/coga-usable/#provide-a-login-that-does-not-rely-on-memory-or-other-
             cognitive-skills-pattern">Making Content Usable for People with Cogntive and Learning Disabilities 4.7.1 Provide a Login that Does Not Rely on Memory or Other Cognitive Skills</a></li>
-            <li><a href="https://rawgit.com/w3c/coga/master/issue-papers/privacy-security.html">Security and Privacy Technologies issue paper from the Cognitive Task Force</a>.</li>
-            <li><a href="https://www.w3.org/TR/webauthn/">WebAuthN specification</a>.</li>
-            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API">Web Authentication API on MDN</a>.</li>
-            <li><a href="https://webauthn.io/">WebAuthN Demo site</a>.</li>
-            <li><a href="https://en.wikipedia.org/wiki/OAuth">OAuth on Wikipedia</a>.</li>
+            <li><a href="https://www.w3.org/TR/webauthn/">WebAuthN specification</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API">Web Authentication API on MDN</a></li>
```

Comment:
This will produce this list of links:

*   WebAuthN specification
*   Web Authentication API on MDN
*   WebAuthN Demo site  
     

I think that we should be grouping the WebAuthN links like:

*   WebAuthN specification
*   WebAuthN Demo site
*   Web Authentication API on MDN  
     

I don't understand the function of this, just seeing an odd ordering.

Reply:
Thanks @mgifford. @giacomo-petri does this suggested order work for you?

Reply:
Yes, it works. I believe the intention was to present a list of references without a specific order. However, it would be even more beneficial if we could rearrange them to enhance the coherence of the reading sequence. 
Additionally, I've addressed another typo in these links.

Reply:
@giacomo-petri the order appears to be the same still?

Reply:
I did it. Not sure what happened. Adjusting right now